### ISSUE TITLE
Allow @SchedulerLock annotation on implementation

### ISSUE THREAD

**Issue by tntim96** [lukas-krecan/ShedLock#12]
Nice tool. I've just been setting it up and noticed my `@SchedulerLock` wasn't taking effect until I moved it to the interface, but my `@Scheduled` annotation is on the implementation method. I can probably live with this, but it does make the code less readable. I'm not sure how Spring manages this, but perhaps this tool could do the same?

The relevant code is in `net.javacrumbs.shedlock.spring.SpringLockConfigurationExtractor.java`:
```Java
SchedulerLock annotation = AnnotationUtils.findAnnotation(method, SchedulerLock.class);
```

**Comment by lukas-krecan-lt** [comment]
Hi, thanks for feedback. It's possible to do, I have just been postponing it. Will take a look at it in next few days.

**Comment by lukas-krecan** [comment]
Did you mean something like this https://github.com/lukas-krecan/ShedLock/pull/13?

**Comment by lukas-krecan** [comment]
Released in 0.9.0

**Comment by tntim96** [comment]
Thanks!
